pandas/tests/io/test_packers.py

```diff
@@ -128,9 +128,8 @@ def test_string_io(self):
         with ensure_clean(self.path) as p:
 
             s = df.to_msgpack()
-            fh = open(p, 'wb')
-            fh.write(s)
-            fh.close()
+            with open(p, 'wb') as fh:
```

Comment:
Do we have a name used more often than not for the file handle? I notice in this change we have two different ones. The last file I touched they were all `as f:` - if that's used more often than not then let's update these to stay consistent